libs/modelviewer/src/ModelViewer.tsx

```diff
@@ -30,25 +32,25 @@ export const FusionModelViewer = (props: PropsWithChildren<FusionModelViewerProp
             props.options?.fallbackComponent ? props.options.fallbackComponent : Message
```

Comment:
A component named Message being used this way is very confusing. Can we name it something more reasonable?

Reply:
As far i can tell, this component is a custom ErrorBoundary that operates with Errors, Warnings and Info messages.
What do you think is a good name?

ErrorBoundary?

Reply:
Added Error Boundary :)